vita3k/mem/src/mem.cpp

```diff
@@ -116,8 +116,8 @@ bool init(MemState &state, const bool use_page_table) {
     const BOOL ret = VirtualProtect(state.memory.get(), state.page_size, PAGE_NOACCESS, &old_protect);
     LOG_CRITICAL_IF(!ret, "VirtualAlloc failed: {}", get_error_msg());
 #else
-    const int ret = mprotect(state.memory.get(), state.page_size, PROT_NONE);
-    LOG_CRITICAL_IF(ret == -1, "mprotect failed: {}", get_error_msg());
+    // const int ret = mprotect(state.memory.get(), state.page_size, PROT_NONE);
```

Comment:
Instead of disabling mprotect for everything except windows, it should be disabled for android only here

Reply:
Ah yes sorry...